lib/MFA.php

```diff
@@ -112,11 +112,11 @@ public function challengeFactor(
      * @param string $code - The verification code sent to and provided by the end user.
     */
 
-    public function verifyFactor(
```

Comment:
We should leave a copy of `verifyFactor` in place so that we don't introduce a breaking change.